apps/examples/st_things/blink/iotbus_util.h

```diff
@@ -0,0 +1,31 @@
+/* ****************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <tinyara/gpio.h>
+#include <iotbus/iotbus_gpio.h>
+
+#define LED_POWER_PORT (45) 
```

Comment:
Don't make it hard-coded. When we change a board, it will not work due to this. Please make it config.
Please refer iotbus_test example.